submissions/nadsatt/friends-app/modules/search-state-keepers.js

```diff
@@ -0,0 +1,33 @@
+import { SearchStateKeeper,
+         FilterStateKeeper } from './state-keepers.js';
```

Comment:
Prettier :)))

Reply:
@zonzujiro  sorry for that. I had some problems with understanding how prettier works 😳. I won't write code without prettier anymore. 

Thanks again for spending your time on this pr!